cmd/maya-apiserver/spc-actions/storagepool_delete.go

```diff
@@ -17,23 +17,32 @@ limitations under the License.
 package storagepoolactions
 
 import (
+	"errors"
 	"fmt"
 	"github.com/golang/glog"
 	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/storagepool"
 )
 
-func DeleteStoragePool(key string) error {
-	// Business logic for deletion of storagepool cr
-	glog.Infof("Storagepool delete event received for storagepoolclaim %s", key)
+func DeleteStoragePool(spcGot *v1alpha1.StoragePoolClaim) error {
+	// Business logic for deletion of storagepool
+	glog.Infof("Storagepool delete event received for storagepoolclaim %s", spcGot.Name)
+
+	casTemplateName := spcGot.Annotations[string(v1alpha1.SPDeleteCASTemplateCK)]
+	if casTemplateName == "" {
+		return errors.New("Aborting storagepool delete operation as no cas template is specified")
```

Comment:
Can the message be rephrased:
"aborting storagepool delete: missing cas template name for delete operation in storagepoolclaim annotations"

Reply:
sure

Reply:
But the first letter of log should be capital only